content/advanced-use.md

```diff
@@ -0,0 +1,104 @@
+# Advanced use
+
+This section of the CLI Guides describes how to make custom configurations to the app's build behavior, testing, and more.
+Ember is designed for a zero config experience for most users, but it was also designed to be extensible. Think of it as configuration without confusion.
+
+## Overall CLI functionality
+
+To begin making configurations, first it is important to understand the overall architecture of what the CLI can do.
+Most custom configuration is done in `ember-cli-build.js`, `environment.js`, and Broccoli plugins.
+
+### Broccoli
+
+Ember uses Broccoli for the build process. Broccoli is an independent project that is similar to tools like webpack and parcel. Common Broccoli configurations include adding assets into the build pipeline. For example, if an app has content in the form of Markdown files that need to be turned into HTML during the build, a Broccoli plugin would be the way to go. This very app you're reading content on right now follows that architecture.
```

Comment:
Are we able to configure broccoli in the same way React users can configure Webpack from scratch?